docs/README.md

```diff
@@ -66,6 +66,37 @@ The Documentation is separated into different parts by sub folders.
 To update the API documentation run the `updateAPI.sh` file.
 Note that you have to install both the Java and Python -doc dependencies.
 
+#### Linux
+
+1. Install [Sphinx](https://pypi.org/project/Sphinx/) and [sphinx_rtd_theme](https://pypi.org/project/sphinx-rtd-theme/).
+
+```bash
+pip install Sphinx
+pip install sphinx-rtd-theme
+```
+
+2. After that, install Python dependencies.
+
+```bash
+pip install numpy pandas py4j
+```
+
+and install any other missing packages.
+
+3. Make sure Maven and Java 8 are installed.
```

Comment:
sure add this as step 4. in the Ubuntu install,
add a new step 5 mark it as optional, and write the call to updateAPI.sh
then move the old step 4 to a step 6